sys/include/random_seed.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2017 Hamburg University of Applied Sciences
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_random
+ *
+ * @{
+ *
+ * @file
+ * @brief       Interface to generate "unique" PRNGs seed
+ *
+ * @author      Peter Kietzmann <peter.kietzmann@haw-hamburg.de>
+ */
+
+#ifndef RANDOM_SEED_H
+#define RANDOM_SEED_H
+
+#include <inttypes.h>
+#include "board.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Set default ADC line for random seed
+ *
+ * @todo    Define by board
```

Comment:
I wouldn't call this a todo: this is a typical default value, so the more intuitive comment would be something like: `@note Override in your board configuration if needed`